root/static/scripts/release-editor/fields.js

```diff
@@ -130,15 +130,18 @@ class Track {
 
         // Convert stuff like 111 into 1:11
 
-        if (/^\d+$/.test(length) && (5 - lengthLength) <= 1) {
-            var minutes, seconds, hours;
+        if (/^\d+$/.test(length) && (6 - lengthLength) <= 1) {
```

Comment:
What if someone type 1212121 which has `7` digits? or any greater number?

Reply:
great question can i just return ?:?? instead, i'm unsure of how to do that. Will just length = ?:?? fix it?

Reply:
Either the four latest digits are possibly `minutes:seconds`, for example `48095312` (as 53 < 60 and 12 < 60), then we can just insert missing colons to separate hours from minutes from seconds.
Or the four latest digits cannot be `minutes:seconds`, for example `48096372` (as 63 ≥ 60 and 72 ≥ 60), then we can just do nothing and it will be considered as a large number of seconds.